Source/DafnyLanguageServer/Workspace/DocumentDatabase.cs

```diff
@@ -180,24 +180,26 @@ IRelocator relocator
       }
 
       var cancellationSource = new CancellationTokenSource();
-      var verifiedDocumentTask = VerifyDocumentIfRequiredAsync(databaseEntry, cancellationSource.Token);
+      var verifiedDocuments = VerifyDocumentIfRequired(databaseEntry, cancellationSource.Token);
       var updatedEntry = new DocumentEntry(
         databaseEntry.Version,
         databaseEntry.ResolvedDocument,
-        verifiedDocumentTask,
+        verifiedDocuments.Select(Task.FromResult).DefaultIfEmpty(databaseEntry.ResolvedDocument).ToTask(cancellationSource.Token).Unwrap(),
```

Comment:
Any chance we might share this code with the above?

Reply:
Is ToTask from a third-party package / an extension?  I'm not familiar with this method.